packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -130,6 +94,68 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         relay(request, msg.sender);
     }
 
+    /// @inheritdoc IFastBridge
+    function prove(bytes memory request, bytes32 destTxHash) external {
+        bytes32 transactionId = keccak256(request);
+        prove(transactionId, destTxHash, msg.sender);
```

Comment:
Let's use named variables syntax here (and in relay/claim) for easier maintainability. This wasn't really required when the overload function was literally a few lines away, but with the function ordering enforced that would be helpful:
```suggestion
        prove({transactionId: keccak256(request), destTxHash: destTxHash, relayer: msg.sender});
```